test/runner/run.js

```diff
@@ -112,8 +110,6 @@ export async function run( {
 				);
 				report.total = total;
 
-				cleanupJSDOM( reportId, { verbose } );
```

Comment:
Why is this not needed anymore? Do we handle cleanup differently?

Reply:
Yes, I've integrated jsdom into the new queue in a way that it gets cleaned up in the exit hook along with everything else.